docs/README.md

```diff
@@ -19,7 +19,133 @@ tbd
+        - action: replace
+          regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
+          replacement: "[$2]:$1"
+          sourceLabels: ["__meta_kubernetes_pod_annotation_prometheus_io_port","__meta_kubernetes_pod_ip"]
+          targetLabel: "__address__"
+        - action: replace
+          regex: (\d+);((([0-9]+?)(\.|$)){4})
+          replacement: "$2:$1"
+          sourceLabels: ["__meta_kubernetes_pod_annotation_prometheus_io_port","__meta_kubernetes_pod_ip"]
+          targetLabel: "__address__"
+        - action: labeldrop
+          regex: "__meta_kubernetes_pod_label_(.+)"
+        - sourceLabels: ["__meta_kubernetes_namespace"]
+          action: replace
+          targetLabel: namespace
+        - sourceLabels: ["__meta_kubernetes_pod_name"]
+          action: replace
+          targetLabel: pod_name
+    ```
+
+Congratulations! You should now be able to see your control plane and data plane metrics in the OpenShift Console. Just go to Observe -> Metrics and try the query `istio_requests_total`.
+
+### Integrating with Kiali
+Integration with Kiali really depends on how you collect your metrics and traces.
+
+#### Integrating with Kiali and the OpenShift monitoring stack
+If you followed [Scraping metrics using the OpenShift monitoring stack](#scraping-metrics-using-the-openshift-monitoring-stack), you can setup Kiali to retrieve metrics from there.
+
+*Prerequisites*
+* User Workload monitoring is [enabled](https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html) and [configured](#scraping-metrics-using-the-openshift-monitoring-stack)
+* Kiali Operator is installed
+
+*Steps*
+1. Create a ClusterRoleBinding for Kiali so it can view metrics from user-workload monitoring
+
+    ```yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: kiali-monitoring-rbac
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cluster-monitoring-view
+    subjects:
+    - kind: ServiceAccount
+      name: kiali-service-account
+      namespace: istio-system
+    ```
+1. Find out the revision name of your Istio instance. In our case it is `test`.
+    
+    ```bash
+    $ kubectl get istiorevisions.operator.istio.io 
+    NAME   READY   STATUS    IN USE   VERSION   AGE
+    test   True    Healthy   True     v1.21.0   119m
+    ```
+1. Create a Kiali resource and point it to your Istio instance. Make sure to replace `test` with your revision name in the fields `config_map_name`, `istio_sidecar_injector_config_map_name`, `istiod_deployment_name` and `url_service_version`.
+
+    ```yaml
+    apiVersion: kiali.io/v1alpha1
+    kind: Kiali
+    metadata:
+      name: kiali-user-workload-monitoring
+      namespace: istio-system
+    spec:
+      external_services:
+        istio:
+          config_map_name: istio-<smcp-name>
```

Comment:
In line 126, we are suggesting the user to replace `test` with the revision name in the fields `config_map_name` (and so on). So, should this actually be 

```
config_map_name: istio-<test>
```

Reply:
Whoops. Taken care of